### ISSUE TITLE
Navbar 'has-navbar-fixed-' attribute on html tag should not be removed on mount

### ISSUE THREAD

**Issue by pm0** [couds/react-bulma-components#243]
When using a fixed Navbar component, the class `has-navbar-fixed-top` or `has-navbar-fixed-bottom` is applied to the html tag. It looks like when the Navbar mounts, if that class exists it's removed and re-added. This causes a short flicker on first load or when navigating to another page in the site, as the padding is applied.

> src/components/navbar/navbar.js

```
const html = window.document.querySelector('html');
html.classList.remove('has-navbar-fixed-top');
html.classList.remove('has-navbar-fixed-bottom');
if (fixed) {
  html.classList.add(`has-navbar-fixed-${fixed}`);
}
```

I'm currently overriding like this in my index.scss but it's applying a global style not set by the Navbar which isn't ideal.

```
html {
  padding-top: $navbar-height;

  &.has-navbar-fixed-top {
    padding-top: $navbar-height;
  }
}
```

**Comment by MrCreeper1008** [comment]
I will look into this. Thank you!

**Comment by MrCreeper1008** [comment]
Fixed in PR #244
